nixos/modules/hardware/video/amdgpu-pro.nix

```diff
@@ -31,17 +35,25 @@ in
 
     boot.extraModulePackages = [ package ];
 
+    boot.kernelPackages =
```

Comment:
@vcunat What do you think of this?  It feels wrong, but this kernel module requires linux 4.10 w/ `KALLSYMS_ALL=y`, and I couldn't find a precedent for influencing the kernel config.

Reply:
First of all, 4.10 branch was killed upstream, and we already deleted it in nixpkgs as well.  Second, just the fact of users having to build their own kernel would be quite a nuisance.  Perhaps it would be good to look into the suitability of having the option in default kernels (say those > 4.9).

Reply:
4.13 is the only maintained branch newer than 4.9 (which is the latest longterm branch).